docs/package.json

```diff
@@ -43,7 +43,10 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
+    "ember-concurrency": "^2.3.7",
```

Comment:
are these still needed?

Reply:
Looks like we're still using timeout: https://github.com/CrowdStrike/ember-aria/pull/20/files#diff-7fd4f04454d74569940c1c05a59a981f89c36393ca5bf847c712eb4c3e9d924bR6

Reply:
is a lot of code for 
```
await new Promise(resolve => setTimeout(resolve, someTimeoutMs));
```

Reply:
Thanks for the tip (I didn't realize that is all it did...duh)

Fix here: https://github.com/CrowdStrike/ember-aria/pull/20/commits/6f2845c5200adf1bdecb98c768ca865bb727087b

Github actions is still failing on me though even though Github Status is green now...not sure what the deal is